joomla/joomla-cms
# [4.0] com_joomlaupdate reference to class HtmlView

Pull Request for Issue # .

stumbled upon this issue while working on https://github.com/joomla/joomla-cms/pull/30171 and https://github.com/joomla/joomla-cms/pull/30168

### Summary of Changes

On of the benefits of working with an IDE like PHPStorm is that it gives me more information about my code then vi edito can do. :-) For instance.. CMD + Click on a reference to head directly to that function in that file. 
If set correctly!

### Testing Instructions

- Open an IDE like PHPStorm and open the changed files. (for instance : default_nodownload.php)
- Try to cmd/ctrl + click on `$this->getModel()`

- all template files from joomlaupdate should reference to the same class HtmlView (if needed)
 Only default_reinstall is set correct. 

### Actual result BEFORE applying this Pull Request

PHPStorm shows a yellow background color to indicate something. When hover HTML it informs you that class HTML cannot be found.

<img width="1234" alt="Schermafdruk 2020-07-24 13 46 13" src="https://user-images.githubusercontent.com/639822/88388642-375f7980-cdb5-11ea-8e47-af70e5994ffc.png">
<img width="1237" alt="Schermafdruk 2020-07-24 13 46 25" src="https://user-images.githubusercontent.com/639822/88388654-3a5a6a00-cdb5-11ea-9218-fcb5dd902019.png"> 
<img width="1234" alt="Schermafdruk 2020-07-24 13 46 40" src="https://user-images.githubusercontent.com/639822/88388656-3af30080-cdb5-11ea-83ab-da0d8013136e.png">

### Expected result AFTER applying this Pull Request

All files reference to the same class HtmlView, if needed... default_updatemefirst.php does not need the reference. 

<img width="1197" alt="Schermafdruk 2020-07-24 13 48 35" src="https://user-images.githubusercontent.com/639822/88388841-90c7a880-cdb5-11ea-95f5-7c499430c44e.png">

### Documentation Changes Required
